my_class.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'my_enumerables'
+
+# Represents a list of elements that pass certain conditions
+class MyList
+  include MyEnumerable
+
+  def initialize(*elements)
+    @list = elements
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] [**OPTIONAL**] It could be better to create your own `each` method as the previous CR mentioned it ♻️  . Your project work properly and give the right output  😄 . According to what he mentioned I will suggest you to test the code below , It will help you to have more insight about the `each` method in ruby 👍 
```bash
class MyList

[...]

def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end

[...]

end
```